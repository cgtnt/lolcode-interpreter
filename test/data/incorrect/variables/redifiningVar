HAI

I HAS A VAR ITZ 2
I HAS A VAR ITZ 3

KTHXBYE
