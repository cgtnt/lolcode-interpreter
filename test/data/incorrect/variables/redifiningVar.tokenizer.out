@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_DECLARE_VAR:I HAS A-@3:T_T_IDENTIFIER:VAR-@3:T_DECLARE_SET_VAR:ITZ-@3:T_T_INT:2-@4:T_COMMAND_TERMINATOR:
-@4:T_DECLARE_VAR:I HAS A-@4:T_T_IDENTIFIER:VAR-@4:T_DECLARE_SET_VAR:ITZ-@4:T_T_INT:3-@6:T_COMMAND_TERMINATOR:
-@6:T_END:KTHXBYE-@7:T_COMMAND_TERMINATOR:
-@7:T_EOF:
